pom.xml

```diff
@@ -732,7 +732,7 @@
                                         <source>${project.basedir}/src/main/311+-nondb/scala</source>
                                         <source>${project.basedir}/src/main/311until330-all/scala</source>
                                         <source>${project.basedir}/src/main/320+/scala</source>
-                                        <source>${project.basedir}/src/main/322+/scala</source>
+                                        <source>${project.basedir}/src/main/322until330/scala</source>
```

Comment:
Why remove 322+?  That forces ShimVectorizedColumnReader to be copied essentially verbatim when it otherwise would not need to be.  Seems like we want both a 322+, to house code that truly is common for everything >= 3.2.2, and a separate 322until320 for code that only applies to Spark >= 3.2.2 and < 3.3.0.

Reply:
Sorry, my mistake! I will restore 322+ and let 322until330 extend 322+.